packages/moonstone/DropDown/DropDown.js

```diff
@@ -0,0 +1,301 @@
+/**
+ * Moonstone styled DropDown components
+ *
+ * @example
```

Comment:
The example is throwing `ReferenceError: DropDown is not defined` in the docs

Reply:
Made a branch in docs `feature/ui-update` and added the import for `DropDown`. Just need to update the `sample-runner` and run `npm run make-runner`.